legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/project/config/ProjectCreationConfiguration.java

```diff
@@ -99,38 +65,14 @@ public int hashCode()
         {
             hash = (31 * hash) + Objects.hashCode(this.artifactIdPattern.pattern());
         }
-        hash = (31 * hash) + this.disallowedTypes.hashCode();
-        hash = (31 * hash) + this.disallowedTypeMessages.hashCode();
         return hash;
     }
 
     @Override
     public String toString()
     {
-        StringBuilder builder = new StringBuilder("<ProjectCreationConfiguration defaultProjectStructureVersion=").append(this.defaultProjectStructureVersion)
-                .append(" groupIdPattern=").append(this.groupIdPattern)
-                .append(" artifactIdPattern=").append(this.artifactIdPattern)
-                .append(" disallowedTypes=[");
-        boolean first = true;
-        for (ProjectType type : this.disallowedTypes)
-        {
-            if (first)
-            {
-                first = false;
-            }
-            else
-            {
-                builder.append(", ");
-            }
-            builder.append(type);
-            String message = this.disallowedTypeMessages.get(type);
-            if (message != null)
-            {
-                builder.append(" (\"").append(message).append("\")");
-            }
-        }
-        builder.append("]>");
-        return builder.toString();
+        return "<ProjectCreationConfiguration defaultProjectStructureVersion=" + this.defaultProjectStructureVersion + " groupIdPattern=" + this.groupIdPattern + " artifactIdPattern=" + this.artifactIdPattern +
+            "]>";
```

Comment:
I don't think you need the closing square bracket here.
```suggestion
            ">";
```
Also, this could go on the previous line.